src/browser_action.js

```diff
@@ -452,6 +452,7 @@ browseraction.createEventDiv_ = function(event) {
     }));
   } else {
     startTimeDiv.css({'background-color': event.feed.backgroundColor});
+    startTimeDiv.attr({'title': event.feed.title}); // Show calendar name on mouseover
```

Comment:
Oh BTW you can just chain this to the previous line
`startTimeDiv.css(…).attr(…);`